src/content/faq/GettingStarted.md

```diff
@@ -47,4 +47,24 @@ Before you start your learning journey, make sure you know [what an API is](http
 
 No, not at all. [GraphQL is a specification](https://spec.graphql.org/) that can be [implemented in any language](/learn/schema/#type-language). Our [Code page](/code/) contains a long list of libraries in many different programming languages to help with that.
 
-It’s understandable why you’d think this, though. GraphQL was introduced at a [React conference](https://www.youtube.com/watch?v=9sc8Pyc51uU) and [GraphQL.js](/graphql-js/) is one of the most widely used implementations to date. We know this can be confusing, so we’re working to improve our documentation and add more code samples that aren’t written in JavaScript.
\ No newline at end of file
+It’s understandable why you’d think this, though. GraphQL was introduced at a [React conference](https://www.youtube.com/watch?v=9sc8Pyc51uU) and [GraphQL.js](/graphql-js/) is one of the most widely used implementations to date. We know this can be confusing, so we’re working to improve our documentation and add more code samples that aren’t written in JavaScript.
+
+### Can I use GraphQL to make graphs or charts?
+
+Yes, graphql2chartjs is an open source tool that reshapes your GraphQL data as per the ChartJS API. This makes building charts as easy as simply making a GraphQL query. The idea behind this tool was to leverage GraphQL's realtime subscriptions to build realtime charts by restructuring the GraphQL data to a form that ChartJS expects and generate ChartJS compliant data object from your GraphQL response by simply adding a few aliases in your GraphQL query.
+
+For more information on graphql2chartjs, check out their [documentation](https://github.com/hasura/graphql2chartjs).
```

Comment:
This seems like a cool library. But I'm a little concerned that it was last updated in 2019 & has a few open issues suggesting that things aren't working. Do you have any experience with it?

Reply:
Well, I haven't worked with this library yet. So do you want me to remove it?